content/source/docs/enterprise/install/installer.html.md

```diff
@@ -12,6 +12,8 @@ installer to set up Terraform Enterprise on a customer-controlled machine.
 
 -> **Important:** Before you begin, consult the [Pre-Install Checklist](../before-installing/index.html) for prerequisites. You'll need to prepare data files and a Linux instance.
 
+-> **Important:** Cloning an already installed Terraform Enterprise host machine can be done with extra cares. It is not allowed to have more than one Terraform Enterprise instances connecting to the same database unless the installation is performed for the [Active/Active architecture](active-active.html). When cloning Terraform Enterprise host machine, these following files contain the private IP address and the hostname of the previous installed environment, `/etc/sysconfig/replicated` or `/etc/default/replicated` and `/etc/sysconfig/replicated-operator` or `/etc/default/replicated-operator`. It is required to ensure that those configuration files maintain the correct private IP address and the hostname of the Terraform Enterprise host machine at the restoration from the image. The `replicated` and `replicated-operator` services should also be restarted when the configuration files are updated.
```

Comment:
Thanks for this addition! I had a couple of clarifying questions. 

This seems like too much information to put into a callout box, since it might be hard for the reader to visually parse. Does the pre-installation checklist previously reference not exist any more, or is there a reason these items shouldn't be a bulleted list in the regular body of the doc? 

I took a guess at some possible edits, but please check them thoroughly; I'm not sure I preserved the meaning. I'm specifically hesitant about a couple of things: 

> at the restoration from the image

Could replace at in the above sentence with "before", "during", or "after"? (I guessed "before").

> services should also be restarted

Does the operator need to manually restart these services, or do they just need to verify that the services have restarted automatically? (I guessed that the operator needed to restart them.)

> when the configuration files are updated

Does the operator update these files or are they automatically updated somehow? (I guessed that the operator updates them.)

```suggestion
-> **Important:** Be careful when cloning an already installed Terraform Enterprise host machine. You cannot have more than one Terraform Enterprise instance connect to the same database unless the installation is uses the [Active/Active architecture](active-active.html). When cloning a Terraform Enterprise host machine, you must make sure that the `/etc/sysconfig/replicated` or `/etc/default/replicated`, and `/etc/sysconfig/replicated-operator` or `/etc/default/replicated-operator` maintain the correct private IP address and the hostname of the Terraform Enterprise host machine before the restoration from the image. You should also restart the `replicated` and `replicated-operator` services when you update the configuration files.
```

Reply:
@judithpatudith as far as I know we currently don't have a `How To` document for cloning an existing VM with TFE pre-installed.

All the existing documents we have for TFE installation are to install TFE from scratch, there are customers already using the clone method and we are aware that Replicated also supports it. I am not certain that this installation method requires a dedicated section or it should be only an additional note on any existing document so at this stage I am just trying to add this note at the top of the installation page as surprisingly some customers have been doing what we never documented and failed then raised support tickets with us. 

Would it make more sense to relocate the note to this file below? I think the pre-install checklist is still something for fresh install rather than cloning VM.

https://github.com/hashicorp/terraform-website/blob/31655560ccc856ee3fc4e2c05b439edf522b1827/content/source/docs/enterprise/install/index.html.md

Reply:
I don't see this is related to Active/Active architecture. The use case could be users clone dev environment into staging or from staging into production